src/shims/x86/mod.rs

```diff
@@ -664,6 +725,30 @@ fn convert_float_to_int<'tcx>(
     Ok(())
 }
 
+/// Calculates absolute value of integers in `op` and stores the result in `dest`.
```

Comment:
Leave a comment about the behaviour when computing the `abs` of the negative most value representable by the type